src/main/steps/applicant1/hub-page/sole/hub.njk

```diff
@@ -24,4 +24,13 @@
   </p>
 {% endset %}
 
+{% set conditionalOrderGrantedDocumentLineHtml %}
```

Comment:
if this is only used in the src/main/steps/applicant1/hub-page/sole/latest-update-content/conditional-order-pronounced.njk file, maybe move it there.